content/shmem_query_thread.tex

```diff
@@ -0,0 +1,34 @@
+\apisummary{
+Returns the level of thread support provided by the library.}
+
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_query_thread(int *provided);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+The \FUNC{shmem\_query\_thread} call returns the level of thread support
+currently being provided. The value returned will be same as \VAR{provided}
+returned in the \FUNC{shmem\_init\_thread}, if the \openshmem{} library was
+initialized by \FUNC{shmem\_init\_thread}.
+}
```

Comment:
Jim: What does `shmem_query_thread` return for library initialized by `shmem_init`? Requesting clarification that specification does not define what the library which (of the four) thread level is returned if `shmem_init` is used to initialize.